config/urls.py

```diff
@@ -26,7 +26,7 @@
 
     # Authentication
     path("login/", LoginView.as_view(template_name='users/home.html'), name='login'),
-    path("logout/", LogoutView.as_view(next_page='/'), name='logout'),
+    path("logout/", LogoutView.as_view(next_page='home'), name='logout'),
```

Comment:
did you check that `home` is working fine? I suggest you check logout functionality once. if you are putting home there.